src/bindings/c/include/openvino/c/ov_infer_request.h

```diff
@@ -161,6 +166,7 @@ ov_infer_request_get_profiling_info(ov_infer_request_t* infer_request, ov_profil
  * @brief Release the memory allocated by ov_profiling_info_list_t.
  * @ingroup infer_request
  * @param profiling_infos A pointer to the ov_profiling_info_list_t to free memory.
+ * @return none
```

Comment:
I think we can remove return for void functions :)

Reply:
sure.